app/src/main/res/layout/main_fragment.xml

```diff
@@ -30,6 +30,15 @@
         android:layout_height="wrap_content"
         android:layout_margin="5dp">
 
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_margin="10dp"
+            android:text="Cases of Covid-19 by Country"
+            android:textColor="#1DA1F2"
+            android:textAlignment="center"
+            android:textSize="16dp" />
```

Comment:
Added title to bar graph for easier comprehension.

Reply:
@holtcl I really appreciate your UI changes, but we are keeping the UI at minimal level. This block of code will become clutter in our codebase. We might change the text view in future.

Reply:
@sukhbirsekhon Good point here.  The goal is to minimize clutter/distractions on the screen, so everything on the screen should have a very good, defined purpose.  Otherwise, remove it.